src/Rating.js

```diff
@@ -145,28 +145,16 @@ Rating.STATE = {
      */
     options: {
         validator: Array.isArray,
-        value: [
-            {
-                value: 1,
-                title: ''
-            },
-            {
-                value: 2,
-                title: ''
-            },
-            {
-                value: 3,
-                title: ''
-            },
-            {
-                value: 4,
-                title: ''
-            },
-            {
-                value: 5,
-                title: ''
-            }
-        ]
+        valueFn: function() {
```

Comment:
Those smarter codes are usually more confusing than beneficial. I would suggest to leave the verbose one for better didactic.